ai/archetypeai/v0/README.mdx

```diff
@@ -12,59 +12,96 @@ It can carry out the following tasks:
 - [Summarize](#summarize)
 - [Upload File](#upload-file)
 
+
+
 ## Release Stage
 
 `Alpha`
 
+
```

Comment:
ℹ️ You'll notice extra newlines added everywhere. I had avoided this at the cost of a less readable template. But it is ok to have ugly auto-generated code (as long as the rendered Markdown / HTML isn't affected). What we want to read clearly is the source template and the rendered documentation.